test/unit/BPool.t.sol

```diff
@@ -610,8 +610,8 @@ contract BPool_Unit_SwapExactAmountIn is BasePoolTest {
 
   function _assumeHappyPath(SwapExactAmountIn_FuzzScenario memory _fuzz) internal pure {
     // safe bound assumptions
-    _fuzz.tokenInDenorm = bound(_fuzz.tokenInDenorm, MIN_WEIGHT, MAX_WEIGHT);
-    _fuzz.tokenOutDenorm = bound(_fuzz.tokenOutDenorm, MIN_WEIGHT, MAX_WEIGHT);
+    _fuzz.tokenInDenorm = bound(_fuzz.tokenInDenorm, MIN_WEIGHT, MAX_WEIGHT / 2);
+    _fuzz.tokenOutDenorm = bound(_fuzz.tokenOutDenorm, MIN_WEIGHT, MAX_WEIGHT / 2);
```

Comment:
could u explain? 👀

Reply:
`MAX_TOTAL_WEIGHT == MAX_WEIGHT`
So if we use `MAX_WEIGHT` as the max `tokenInDenorm` is not realistic because it will sum up more than the total weight permitted when adding `totalOutDenorm`.
I divide it to give some safe space to the other denorm.
wdyt?

Reply:
hmmmm but then it'd be between `min_weight` and `total_max - min_weight`? not sure `/2` covers all cases